tests/user_examples/extra_src/shared/general_continuum/continuum_dynamics/continuum_dynamics_complex.hpp

```diff
@@ -70,5 +70,125 @@ void BaseShearStressRelaxation2ndHalfWithWall<BaseShearStressRelaxation2ndHalfTy
     }
     this->velocity_gradient_[index_i] += velocity_gradient;
 }
+
+//============================================Plasticity===========================================//
+//=================================================================================================//
+//============BaseStressRelaxation1stHalfWithWall & BaseStressRelaxation2ndHalfWithWall============//
+//=================================================================================================//
+template <class BaseStressRelaxation1stHalfType>
+Vecd BaseStressRelaxation1stHalfWithWall<BaseStressRelaxation1stHalfType>::computeNonConservativeAcceleration(size_t index_i)
+{
+    return this->acc_prior_[index_i];
+}
+
+template <class BaseStressRelaxation1stHalfType>
+void BaseStressRelaxation1stHalfWithWall<BaseStressRelaxation1stHalfType>::interaction(size_t index_i, Real dt)
+{
+    BaseStressRelaxation1stHalfType::interaction(index_i, dt);
+
+    Vecd acc_prior_i = computeNonConservativeAcceleration(index_i);
+    Vecd acceleration = acc_prior_i;
+    Real rho_dissipation(0);
+
+    Matd stress_tensor_i = this->reduceTensor(this->stress_tensor_3D_[index_i]);
+
+    for (size_t k = 0; k < fluid_dynamics::FluidWallData::contact_configuration_.size(); ++k)
+    {
+        StdLargeVec<Vecd>& acc_ave_k = *(this->wall_acc_ave_[k]);
+        Neighborhood& wall_neighborhood = (*fluid_dynamics::FluidWallData::contact_configuration_[k])[index_i];
+        for (size_t n = 0; n != wall_neighborhood.current_size_; ++n)
+        {
+            size_t index_j = wall_neighborhood.j_[n];
+            Vecd& e_ij = wall_neighborhood.e_ij_[n];
+            Real dW_ijV_j = wall_neighborhood.dW_ijV_j_[n];
+            Real r_ij = wall_neighborhood.r_ij_[n];
+            Vecd nablaW_ijV_j = wall_neighborhood.dW_ijV_j_[n] * wall_neighborhood.e_ij_[n];
+
+            Real face_wall_external_acceleration = (acc_prior_i - acc_ave_k[index_j]).dot(-e_ij);
+            Real p_in_wall = this->p_[index_i] + this->rho_[index_i] * r_ij * SMAX(Real(0), face_wall_external_acceleration);
+            Matd stress_tensor_in_wall = stress_tensor_i;
```

Comment:
Why introduc stress_tensor_in_wall? not directly use stress_tensor_i?

Reply:
delete stress_tensor_in_wall